pydy/tests/test_bodies.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+# external libraries
+from sympy import Symbol
+from sympy.physics.vector import Point, Dyadic
+from sympy.physics.mechanics import ReferenceFrame
+
+# local
+from ..bodies import Body, Ground
+from ..force import Force
+
+class TestBody():
+    def setup(self):
+        self.body = Body('body')
+
+    def test_body_init(self):
+        assert self.body.name == 'body'
+        assert self.body.parent is None
+        assert self.body.child is None
+        assert isinstance(self.body.masscenter, Point)
+        assert isinstance(self.body.frame, ReferenceFrame)
+        assert isinstance(self.body.mass, Symbol)
+        assert len(self.body.inertia) == 2
+        assert isinstance(self.body.inertia[0], Dyadic)
+        assert isinstance(self.body.inertia[1], Point)
+
+    def test_body_properties_value(self):
+        name = self.body.name
+        assert self.body.masscenter == Point(name + '_MassCenter')
+        assert self.body.frame == ReferenceFrame(name + '_ReferenceFrame')
+        assert self.body.mass == Symbol(name + '_Mass')
+
+    def test_body_add_force(self):
+        assert isinstance(self.body.force_list, list())
+        assert len(self.body.force_list) == 0
+
+        self.body.add_force(Force.GRAVITY)
```

Comment:
A force is a vector applied to a point. That's the big generalization. I think it would be fine to create `class Load(object)`, `class Force(Load)`, and `class Torque(Load)` (not sure if Load is necessary) classes. But maybe these classes should somehow be smart enough to apply equal and opposite forces. You could initalize them like `Force(vec, point1, opp_point=point2)` or something.